third_party/move/move-compiler-v2/tests/checking/inlining/shadowing_unused.exp

```diff
@@ -61,8 +61,8 @@ module 0x42::Test {
           }
         }
     }
-    spec fun $foo(f: |(u64, u64)|(),z: u64);
-    spec fun $quux(f: |(u64, u64)|(),_z: u64);
+    spec fun $foo(f: |(u64, u64)|,z: u64);
```

Comment:
Please file a bug to add a space after comma in these specs.

Reply:
For the debug output?